TheDeafOne/ScheduLite
# Kw/front end security management

set up signup page, login page, and profile routing. 
right now it's mega ugly, but that'll have to wait till we restructure CSS usage.

To test:
start the webservice and clientapp
navigate to /signup and make a user
navigate to /signin and sign in as the user you just made
reloading the browser on /signin should automatically take you to the user profile (which just says hello rn)
